infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -353,7 +366,8 @@ private void bootstrapPrimarySequencerServer() {
      * @return True if recovery was successful. False otherwise.
      */
     private boolean runRecoveryReconfiguration() {
-        Layout layout = new Layout(serverContext.getManagementLayout());
+        Layout layout = getManagementLayout();
+        Layout localRecoveryLayout = new Layout(layout);
```

Comment:
getManagementLayout returns null, but the copy constructor doesnt.

Layout(@Nonnull Layout layout)